specification/components/r4/schemas/responses/requestUploadUri/400Response.yaml

```diff
@@ -0,0 +1,20 @@
+description: |
+  Where status code 400 (Bad Request) is returned then an [NHSDigital-OperationOutcome](https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome) will be included in the body, as detailed below.
+  Check diagnostics property for specific information regarding the error.
+  
+  | issue.details.coding.code | issue.code   | Description                                           |
+  | ------------------------- | ------------ | ----------------------------------------------------- |
+  | REC_BAD_REQUEST           | required     | The service request is not in a valid state.          |
```

Comment:
The description included here should be more generic, perhaps something like "The server detected an issue whilst validating the supplied request".

Reply:
Ah I see this description is specific to the request upload URI, in that case this description is probably fine.